packages/ui-components/src/components/radio-list/radio-list.scss

```diff
@@ -0,0 +1,10 @@
+@import '../../assets/styles/globals';
+
+.radio-list-container {
+	display: flex;
+	flex-direction: column;
+
+	.radio-list-item {
+		margin-bottom: $spacing-2x;
```

Comment:
You can use `gap: $spacing-2x` on the `.radio-list-container` to avoid the last `.radio-list-item` having an unnecessary margin-bottom.

Reply:
that would cause a 4px difference since form-label already has 4px margin bottom, but i'll change it to something simpler :)